doc/mgr/dashboard.rst

```diff
@@ -429,6 +429,13 @@ the host name:
 
    ceph dashboard set-rgw-api-ssl-verify False
 
+If you want to set a custom hostname or IP to resolve the rgw endpoint, you can
+use the RGW_HOSTNAME setting to do it.
```

Comment:
To set a custom hostname or address for an RGW endpoint, set the value of ``RGW_HOSTNAME`` accordingly:

Reply:
fyi, made a few more changes according to the recent modification but they are minor updates. PTAL, thanks